src/main/scala/com/databricks/sparkdl/ImageUtils.scala

```diff
@@ -72,15 +90,40 @@ private[sparkdl] object ImageUtils {
     image
   }
 
+  /** Returns the number of channels in the passed-in buffered image. */
```

Comment:
This method and `getOCVType` invert the conversion done in [ImageSchema](https://github.com/apache/spark/blob/1edb3175d8358c2f6bfc84a0d958342bd5337a62/mllib/src/main/scala/org/apache/spark/ml/image/ImageSchema.scala#L146)